apps/Test App/Test App/Screens/UploadListView.swift

```diff
@@ -23,6 +23,12 @@ struct UploadListScreen: View {
     }
 }
 
+extension MuxUpload {
+    var objectIdentifier: ObjectIdentifier {
```

Comment:
In re the PR description, if a client wants a cheap way to check for equality, `ObjectIdentifier` should be what they use.